content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,39 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. The
+workspace is the directory containing _user visible_ part of your
+<abbr>project</abbr> e.g. raw datasets, source code, ML models, etc. Users work
+in this directory using their data and model files that and manipulate the
+contents through DVC commands.
+
+Files and directories in the workspace can be added to DVC (`dvc add`) or they
+can be downloaded from external sources (`dvc get`, `dvc import`,
```

Comment:
> `dvc get`, `dvc import`, `dvc import-url`

Let's just list "import, get, etc." I think.

Reply:
I repeat `dvc` part for links. Is there a way to link implicitly without `dvc`? Otherwise I'll link them manually.

Reply:
Yes, def. keep `dvc`. I meant basically to remove the `import-url` link (kind of redundant with `import`).

Reply:
But actually `add` can also download data to the workspace (see `--out` and `--to-remote` options). Also, `import*` commands download AND track data. You may want to rephrase this part accordingly 🙂

Reply:
Sometimes I think there is no command that hasn't got a duplicate, somehow :) I try to mention commands in passing, if we'd consider each and every option to commands, we'll need to duplicate the command reference here IMHO. 

We may just delete the commands if you would like.

Reply:
I can also list all possibilities for each functionality, like

In the workspace, you can
- Import files and directories (`dvc add --out`, `dvc import-url`...)

but I think this will turn the document into a list of commands and options.

Reply:
No need to list every command usage of course, agreed!

My point was that these 3 commands mentioned actually overlap in a way that makes the current text slightly incorrect. In any case, the main use case of `add` is not to "add" but to "track", actually. Please check each cmd ref to try to find the right terms when needed 🙂 

"Download" is correct for `get/import` but `add` can also download (and they can all "transfer") so I'd avoid that term probably. And in fact I wouldn't even mention `get` here, since it doesn't require a DVC project/workspace. For `import` I'd try to use the cmd name as the relevant action (to "import") I guess...

Reply:
> the main use case of add is not to "add" but to "track"
> For import I'd try to use the cmd name as the relevant action (to "import")

then again `import*` also track the downloaded data 😅. Maybe it should be a single sentence about tracking and put both `add`, `import`, `import-url` in the same parenthesis.